### PR TITLE
add support for dark mode

### PR DESCRIPTION
**Pull request by rossPatton**
addresses #250 

not exactly the best, but hey, it works

for the svg icons, the reason i  embedded them directly so i could take advantage of css classes to change the fill based on theme

### CODE REVIEW

File: chapter_local/nyc.md [hunk 1]
```diff
@@ -50,6 +37,16 @@ permalink: /nyc/
 
 <hr />
 
+<div class="flex justify-between ai-center pad">
```
Comment (shushugah) [inline chapter_local/nyc.md:40]: `ai-center` is such an intriguing name!

**Review by shushugah — APPROVED** [review]
The preview and work looks amazing! I noticed that there is a cascading css rule for $black variable in same file

File: _sass/_variables.scss [hunk 2]
```diff
@@ -1,7 +1,9 @@
 $red: rgb(232, 68, 46);
-$black: rgb(33, 33, 33);
-$white: rgb(255, 255, 255);
-$grey: #333;
+$black: #000;
+$white: #fff;
+$grey: #222;
+$grey2: #444;
+$grey3: #f6f6f6;
 $black: rgb(2, 14, 15);
```
[the remark this answers is not in the document]
Comment (rossPatton) [reply _sass/_variables.scss:7]: hmm, i'd rather go with line 2 personally. not much difference either way

File: chapter_local/nyc.md (hunk 1 above)
Comment (rossPatton) [reply chapter_local/nyc.md:40]: atomic class names are just descriptive. sometimes i see this as items-center, but that's all it is. it **a**ligns **i**tems along the vertical center

Comment (rossPatton) [reply chapter_local/nyc.md:40]: i'd rather have tailwind just auto-generate all these and never think about css again, personally. for a future version maybe

**Pull request merged**